pkg/depository/create.go

```diff
@@ -62,6 +63,10 @@ func NewCreateDepositoryCmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
+			walletDir, err := cmd.Flags().GetString("wallet-dir")
```

Comment:
keep same flag name 
https://github.com/bestchains/bc-cli/blob/main/pkg/account/get.go#L67

Reply:
I mean you should use `wallet` instead of `wallet-dir`